target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -938,12 +939,13 @@ contract EntropyTest is Test, EntropyTestUtils, EntropyEvents {
         );
     }
 
+    // TODO: restore this test once providers have custom gas limits, which allow us to toggle between
+    // the old and new failure behavior.
```

Comment:
(restoring this test will ensure backward compatibility of the flow, per the comment above)